### ISSUE TITLE
Adding to notes while reading pdf

### ISSUE THREAD

**Issue by keithcdodd** [fonol/anki-search-inside-add-card#70]
(1) I would like to be able to take/create notes from this addon on a pdf that I have open and am reading in this addon. However, I notice that when I create a note on a pdf, I can no longer manipulate the pdf pane, and can only work with the new note pane (whether or not I click on/off each window). I apologize if I am missing something. What's the best way to create/add notes while reading a pdf?

(2) While a pdf is open, how can I easily access any/all notes related to that pdf? I can create a new note, but I cannot figure out how to access and edit an already created note while that pdf is open.

Thanks!

**Comment by fonol** [comment]
Hi,
(1) Adding (Anki) notes while reading is like the main idea of the pdf part of the add-on, so that should definitely work. I am not completely sure I get where the problem is, so let's go through it step by step:
1. If you open a pdf note, or click on the book icon to read the first item in the queue, the pdf viewer should open, with the fields to create Anki notes on the left of the window, right? It should look something like this 
https://imgur.com/2Uvfhb3.png
If that's not what you see, something is going terribly wrong, and you could check what happens if you 
   1.1 maximize the Add dialog window
   1.2 press Ctrl+F
2. You can now interact with the pdf, and create notes by filling the fields on the left side.
Is that the part where your problem occurs?

(2) There is nothing yet where you can browse your notes while the pdf is open, such a feature would be cool, but also difficult UI-wise, as place is limited.

**Comment by keithcdodd** [comment]
Hello, sorry that I was not very clear.

1. This works for me. I can open a pdf and see the pdf viewer.
2. I can create flashcards in the normal add dialog for anki on the left-hand side. 

But, I want to create notes on the pdf while reading it (like write down comments and thoughts and summaries and save figure photos to then potentially make flashcards from them later).

My issue comes from the following steps:
1. Open pdf note as you described (great)
2. Click on the "pencil-icon Note" button on the bottom of the pdf pane -> opens new window to make a note (great)
3. But as long as that new window for creating a notes is open, I cannot manipulate or scroll through or do anything with the pdf pane. So, I cannot effectively take notes as I read through the pdf.

For clarification for the "browse notes related to pdf" I personally wouldn't mind if I didn't too easily see the flashcards I made from the pdf, but it would be cool to easily see all the "pencil-icon Notes" I made while reading that pdf.

Thanks!
